hisprite.s

```diff
@@ -66,8 +68,8 @@ SPRITEPTR_H		= $1c
 main:
 	jsr EnableHires
 
-	lda #$00
-	jsr VenetianFill
+	;lda #$00
+	;jsr VenetianFill
```

Comment:
I can't build the ProDOS version so I loaded some data at $2000 outside of the demo program so I could see the sprites moving over an image